lib/attrsets.nix

```diff
@@ -1161,6 +1164,37 @@ rec {
     ```
     :::
 
+    :::{#map-attrs-recursive-example-list .example}
+    # Map over leaf attributes with lists
```

Comment:
This example doesn't match for `mapAttrsRecursive` anymore

Reply:
:facepalm: